test/Libraries/DynamoPythonTests/PythonEditTests.cs

```diff
@@ -21,7 +23,6 @@ protected override void GetLibrariesToPreload(List<string> libraries)
         {
             libraries.Add("DesignScriptBuiltin.dll");
             libraries.Add("DSCoreNodes.dll");
-            libraries.Add("DSIronPython.dll");
```

Comment:
Are you sure you can remove this? I see some test that seem to use IronPython, like the one that uses `IronPythonInfo_TestGraph.dyn`.

Reply:
I believe it should be removed since these tests should be loading the IronPythonExtension which should load DSIronPython anyway, trying to load it twice just results in time spent catching exceptions during each test's startup.

Reply:
these all pass locally.

Reply:
Right because IronPythonExtension is loaded by default..